sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/objects/objects.scala

```diff
@@ -383,15 +383,26 @@ case class Invoke(
       """
     }
 
+    val evalFunc = if (propagateNull) {
+      s"""
+        ${ev.isNull} = $resultIsNull;
+        if (!${ev.isNull}) {
+          $evaluate
+        }
+      """
+    } else {
+      s"""
+        ${ev.isNull} = false;
+        $evaluate
+      """
+    }
```

Comment:
Do we need this? When `propagateNull` is false, in `prepareArguments` `resultIsNull` will be set to false literal.

Reply:
Ur, I missed that and you're right. I'll revert this. Thanks!